lib/sport_ngin_aws_auditor/ec2_instance.rb

```diff
@@ -16,17 +16,27 @@ def get_instances(client=AWS.ec2, tag_name=nil)
       def get_reserved_instances(client=AWS.ec2)
         client.describe_reserved_instances.reserved_instances.map do |instance|
           next unless instance.state == 'active'
-          new(instance, nil, instance.instance_count)
+          new(instance, nil, ec2_classic_support(client), instance.instance_count)
         end.compact
       end
 
       def get_retired_reserved_instances(client)
         client.describe_reserved_instances.reserved_instances.map do |instance|
           next unless instance.state == 'retired'
-          new(instance, nil, instance.instance_count)
+          new(instance, nil, ec2_classic_support(client), instance.instance_count)
         end.compact
       end
 
+      def ec2_classic_support(client=AWS.ec2)
+        attr_vals = client.describe_account_attributes.account_attributes.first.attribute_values
```

Comment:
Could we be more specific here and inspect the values for `supported-platforms` attribute?